pkgs/top-level/all-packages.nix

```diff
@@ -13703,6 +13703,13 @@ in
 
   minio = callPackage ../servers/minio { };
 
+  mkchromecast = callPackage ../applications/networking/mkchromecast {
+    alsaSupport    = config.mkchromecast.alsaSupport or false;
+    systemTray     = config.mkchromecast.systemTray or true;
+    ffmpegSupport  = config.mkchromecast.ffmpegSupport or true;
+    youtubeSupport = config.mkchromecast.youtubeSupport or true;
```

Comment:
I'm not very familiar with the `config` system. However, I don't think these are useful as they can be achived using a simple `override`.

Reply:
Why did you just mark this as resolved?

Reply:
My bad! I did actually remove it, but some rebasing later it was back.